api/tests/routes/native/v1/account_test.py

```diff
@@ -1301,20 +1301,21 @@ def test_send_phone_validation_code(self, client, app):
 
         assert int(app.redis_client.get(f"sent_SMS_counter_user_{user.id}")) == 1
 
-        token = users_models.Token.query.filter_by(userId=user.id, type=users_models.TokenType.PHONE_VALIDATION).first()
-
-        assert token.expirationDate >= datetime.utcnow() + timedelta(hours=10)
-        assert token.expirationDate < datetime.utcnow() + timedelta(hours=13)
+        encoded_token = app.redis_client.get(token_utils.SixDigitsToken._get_redis_key(token_utils.TokenType.PHONE_VALIDATION, user.id))  # type: ignore [attr-defined]
+        assert encoded_token is not None
+        token = token_utils.SixDigitsToken.load_without_checking(encoded_token, token_utils.TokenType.PHONE_VALIDATION, user.id)  # type: ignore [attr-defined]
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏mypy ne check pas les fichiers de tests, mais pourquoi tu as une erreur ici ?

Reply:
ah non je n'ai pas d'erreur j'ai juste oublié que mypy ne check pas les tests quand je codais